x/gov/keeper/deposit.go

```diff
@@ -162,6 +162,52 @@ func (keeper Keeper) AddDeposit(ctx sdk.Context, proposalID uint64, depositorAdd
 	return activatedVotingPeriod, nil
 }
 
+// SendMinDepositsOfProposalToCommunityPool will sends the min deposits of proposal to community pool
+// from proposer deposits and updated the deposit store with updated deposit amount.
+func (keeper Keeper) SendMinDepositsOfProposalToCommunityPool(ctx sdk.Context, proposalID uint64, proAddr string) error {
```

Comment:
How about the following naming?
```suggestion
func (keeper Keeper) SendProposalMinDepositToCommunityPool(ctx sdk.Context, proposalID uint64, proAddr string) error {
```